data/json/vitamin.json

```diff
@@ -3,39 +3,64 @@
     "id": "calcium",
     "type": "vitamin",
     "name": "Calcium",
-    "min": -600,
-    "deficiency": [ [ "brittlebone", -300 ] ]
+    "deficiency": "hypocalcemia",
+    "min": -720,
+    "disease": [
+      [ -288, -336 ],
+      [ -337, -384 ],
+      [ -385, -720 ]
```

Comment:
> min and max are the lowest/highest levels a vitamin can be set to.

Then proper names for `disease` range IMHO would be `disease_min` and `disease_max`.

> This isn't how the code functions. Each disease entry sets an increasing intensity for the deficiency status.

I saw it. It's rather fragile. You make an assumption that number of levels in `vitamin` and in `effect` for diseases are equal (which others most likely will be unaware of. Besides, it's inconvenient). I'd call `get_max_intensity()` from diseases and use the value to split the range.